Tasks/google-play-release/task.json

```diff
@@ -119,7 +128,41 @@
             "defaultValue": "",
             "groupName": "advanced",
             "required": false,
-            "helpMarkDown": "Paths to additional APK files you want to publish to the specified track (e.g. an x86 build). Wildcards can be used. For example, _\\*\\*/\\*.apk_ to match the first APK file, in any directory. Supports multiple lines or minimatch patterns."
+            "helpMarkDown": "Paths to additional APK files you want to publish to the specified track (e.g. an x86 build). Wildcards can be used. For example, _\\*\\*/\\*.apk_ to match all APK files, in any directory."
+        },
+        {
+            "name": "versionCodeFilterType",
+            "type": "pickList",
+            "label": "Replace version codes",
+            "defaultValue": "all",
+            "groupName": "advanced",
+            "required": false,
+            "helpMarkDown": "Specify version codes to replace in the selected track with the new APKs: all, the comma separated list, or a regilar expression pattern.",
+            "options": {
+                "all": "All",
+                "list": "List",
+                "expression": "Regular expression"
+            }
+        },
+        {
+            "name": "replaceList",
+            "type": "string",
+            "label": "Version code list",
+            "defaultValue": "",
+            "groupName": "advanced",
+            "required": true,
+            "helpMarkDown": "The comma separated list of APK version codes to be removed from the track with this deployment.",
+            "visibleRule": "versionCodeFilterType = list"
+        },
+        {
+            "name": "replaceExpression",
+            "type": "string",
+            "label": "Version code pattern",
+            "defaultValue": "",
+            "groupName": "advanced",
+            "required": true,
+            "helpMarkDown": "The regular expression pattern to select a list of APK version codes to be removed from the track with this deployment, e.g. .*12?(3|4)?5 .",
```

Comment:
You can put the regex string in `` so it is rendered differently and will be clear.